packages/grafana-data/src/types/panel.ts

```diff
@@ -133,6 +146,17 @@ export class PanelPlugin<TOptions = any> extends GrafanaPlugin<PanelPluginMeta>
     this.customFieldConfigs = registry;
     return this;
   }
+
+  /**
+   * Enables configuration of panel's default field config
+   */
+  setFieldConfigDefaults(defaults: FieldConfig) {
+    this.fieldConfigDefaults = {
+      defaults,
+      overrides: [],
+    };
+    return this;
```

Comment:
a bit of a mismatch here property name & function name (in terms of type/param). Not sure what to with overrides, separate function & property? or change argument here to be FieldConfigSource

Reply:
@torkelo updated the API to accept FieldConfigSource. @ryantxu do you have eny thoughts on this if overrides should be also allowed to be set as panel's defaults?